app/views/devise/sessions/new.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="col-md-4 col-md-offset-4">
     <div class="panel panel-default">
       <div class="panel-heading">
-        <h3 class="panel-title">Log In</h3>
+        <h3 class="panel-title"><b>Log In</b></h3>
```

Comment:
This is bad. Avoid stacking similar styles like this for a non-specific purpose. If you think that bold is absolutely necessary, update the `h3` style in some custom CSS file. Regardless, remove the `<b>` tag.